apps/cli/src/command/diff.command.ts

```diff
@@ -158,14 +158,16 @@ export const DiffResourceTask = (
 
 export const DiffCommand = new BackendCommand<DiffOptions>(
   'diff',
-  'Show the difference between local and backend configurations',
+  'show differences between the local and the backend configurations',
+  'Compare the configuration in the specified file(s) with the backend configuration',
```

Comment:
Only curious to know why the first sentence needs to be lower cased

Reply:
By default the library we use, commander, uses lower case when showing the summary of a command/flag, like the in-built help command/version flag.

I just followed the convention.

The alternative was to use custom help and version commands which seemed unnecessary.